javalin/src/main/java/io/javalin/http/JavalinServlet.kt

```diff
@@ -52,6 +53,9 @@ class JavalinServlet(val config: JavalinConfig) : HttpServlet() {
                     if (config.inner.resourceHandler?.handle(wrappedReq, JavalinResponseWrapper(rawRes, rwc)) == true) return@tryWithExceptionMapper
                     if (config.inner.singlePageHandler.handle(ctx)) return@tryWithExceptionMapper
                 }
+                if (type == HandlerType.OPTIONS && isCorsEnabled(config)) { // CORS is enabled, so we return 200 for OPTIONS
+                    return@tryWithExceptionMapper
+                }
```

Comment:
Without this, standard routes would get a 404 even when passed through the handler